src/css/addItem.module.css

```diff
@@ -0,0 +1,372 @@
+    align-items: center;
+    margin-top: 20px;
+    margin-bottom: 20px;    
+
+}
+
+
+.emptyOutfitText {
+  text-align: center;
+  font-size: 1.2rem;
+  margin-top: 2rem;
+  color: #777;
+}
+
+
+/*Item card*/
+
+.itemsGrid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr); /* 4 equal-width columns */
+  gap: 18px; /* space between items (adjust as needed) */
+  justify-items: center; /* center items horizontally */
+  align-items: center;   /* center items vertically */
+}
+
+.itemCard {
+  width: 76px;
+  height: 94px;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 10px;
+  position: relative
+}
+
+.itemImage {
+  width: 62px;
+  height: 72px;
+  object-fit: contain; /* prevent distortion */
+}
+
+/* When selected */
+.selected {
+  border: 2px solid #4CAF50; /* green border */
+}
+
+.checkmarkIcon {
+  position: absolute;
+  top: 5px;      /* Distance from the top edge of the container */
+  right: 5px;    /* Distance from the right edge of the container */
+  width: 9px;
+  height: 7px;   /* fix typo here: '7x' → '7px' */
+  pointer-events: none; /* so clicks pass through */
+}
+
+
+.addItemSection {
+  display: flex;
+  align-items: center;
+  gap: 16px; /* space between button and card */
+  margin-top: 20px; /* add some spacing from above content */
+}
+
+.selectedItemCard {
+  position: relative;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  padding: 8px;
+  width: 120px;
+  text-align: center;
+  background: white;
+  box-shadow: 0 2px 6px rgba(0,0,0,0.1);
+}
+
+.removeItemButton {
+  position: absolute;
+  top: 4px;
+  left: 4px;
+  background: transparent;
+  border: none;
+  font-size: 18px;
+  cursor: pointer;
+  color: #999;
+  line-height: 1;
+}
+
+.removeItemButton:hover {
+  color: #333;
+}
+
+.itemImage {
+  width: 100%;
+  height: auto;
+  border-radius: 4px;
+}
+
+
+/*Styling for board*/
+.outfitBoard {
+  position: relative;
+  width: 100%;
+  height: 400px; /* Adjust as needed */
+  border: 2px solid #ccc;
+  background: #f9f9f9;
+  overflow: hidden;
+}
+
+.outfitItem {
+  position: absolute;
+  width: 50%; /* smaller size */
+  cursor: grab;
+  user-select: none;
+  z-index: 10;
+}
+
+.outfitItem:active {
+  cursor: grabbing;
+}
+
+.selectedItem img {
+  width: 100%;
+  height: auto;
+  pointer-events: none;
+  display: block;
+}
+
+.selectedItem p {
+  font-size: 10px;
+  text-align: center;
+  margin-top: 2px;
+  user-select: none;
+}
+
+
+.homeControls {
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  margin-top: 20px;
+}
+
+.selectedItemsPreview {
+  display: flex;
+  gap: 12px;
+  flex-wrap: wrap;
+}
+
+.selectedPreviewCard {
+  position: relative;
+  width: 100px;
+  border: 1px solid #ccc;
+  border-radius: 6px;
+  padding: 8px;
+  background: #fafafa;
+  text-align: center;
+  user-select: none;
+}
+
+.selectedPreviewImage {
+  width: 80px;
+  height: auto;
+  pointer-events: none;
+}
+
+.removeButton {
```

Comment:
If I am scrolling through here, what is the difference between .removeButton and .removeItem__Button?